Source/WebCore/rendering/RenderFrameSet.cpp

```diff
@@ -188,26 +189,26 @@ void RenderFrameSet::layOutAxis(GridAxis& axis, std::span<const Length> grid, in
     for (int i = 0; i < gridLen; ++i) {
         // Count the total length of all of the fixed columns/rows -> totalFixed
         // Count the number of columns/rows which are fixed -> countFixed
-        if (grid[i].isFixed()) {
-            gridLayout[i] = std::max(grid[i].intValue(), 0);
+        if (grid[i].unit == HTMLDimensionsListValue::Unit::Absolute) {
+            gridLayout[i] = std::max(static_cast<int>(grid[i].number), 0);
```

Comment:
Will this work?
```suggestion
            gridLayout[i] = std::max<int>(grid[i].number, 0);
```
Is the version with a cast better? Is it best to truncate rather than rounding?

Reply:
This whole function is going to be replaced, so, since truncation preserves the existing behavior, I'm going to keep it and re-evaluate in the replacement.